src/Entity/Size.php

```diff
@@ -0,0 +1,156 @@
+<?php
+
+namespace App\Entity;
+
+use App\Repository\SizeRepository;
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Entity(repositoryClass=SizeRepository::class)
+ */
+class Size
+{
+    /**
+     * @ORM\Id
+     * @ORM\GeneratedValue
+     * @ORM\Column(type="integer", name="bm_id")
+     *
+     * @var int id
+     */
+    private int $id;
```

Comment:
Codacy found an issue: [Avoid variables with short names like $id. Configured minimum length is 3.](https://app.codacy.com/gh/CarolineDirat/P7APIBileMo/pullRequest?prid=6297993)